cli/utils.go

```diff
@@ -26,7 +26,7 @@ var (
 	// Status query parameter.
 	Status string = ""
 	// ConfigPath config path parameter.
-	ConfigPath string = ""
+	ConfigPath string = "cli/config.toml"
```

Comment:
Change default path to `./config.toml`, Ideally `config.toml` should present in the same directory (next to) `mainflux-cli`,

Reply:
according to https://github.com/golang-standards/project-layout I would say it should be in `configs/cli/config.toml` or `configs/cli.toml`

Reply:
I have put it in `configs/cli/config.toml`